pageserver/src/tenant/timeline.rs

```diff
@@ -3534,11 +3518,6 @@ impl Timeline {
         }
     }
 
-    async fn flush_frozen_layers_and_wait(&self, at_lsn: Lsn) -> Result<(), FlushLayerError> {
-        let token = self.flush_frozen_layers(at_lsn)?;
-        self.wait_flush_completion(token).await
-    }
-
```

Comment:
So this was inlined because it had three jobs, 2 callsites:
- freeze
- signal that we've flushed
- wait for flush

so now there's less methods:
- one which freezes and signals
- one which waits

You are correct, it was not really necessary to remove this, however, `flush_frozen_layers_and_wait` didn't freeze, it only signalled, and I thought it was stupid.